bin/epaa.py

```diff
@@ -1136,7 +1140,7 @@ def __main__():
             complete_df['wt ligand score'] = complete_df.apply(lambda row: create_ligandomics_column_value_for_result(row, lig_id, 0, True), axis=1)
             complete_df['wt ligand intensity'] = complete_df.apply(lambda row: create_ligandomics_column_value_for_result(row, lig_id, 1, True), axis=1)
     # write mutated protein sequences to fasta file
-    if args.fasta_output:
+    if args.fasta_output and not no_predictions:
```

Comment:
Maybe "successful_predictions" or some name that is TRUE instead of "no_predictions" so we don't have double negative logic here